src/views/home/components/report-issue-button/report-issue-button.view.jsx

```diff
@@ -16,6 +16,15 @@ function ReportIssueButton () {
       >
         Report an issue
       </a>
+      <span className={classes.separator}> | </span>
```

Comment:
Are they blank spaces needed if it already has padding?

Reply:
Correct, not needed. Removed.